ProjFS.Mac/PrjFSKextTests/KauthHandlerTests.mm

```diff
@@ -1,10 +1,17 @@
 #include "../PrjFSKext/kernel-header-wrappers/vnode.h"
-#import <XCTest/XCTest.h>
 #include "../PrjFSKext/KauthHandlerTestable.hpp"
+#include "../PrjFSKext/PerformanceTracing.hpp"
+#import <XCTest/XCTest.h>
 #include "MockVnodeAndMount.hpp"
+#include "MockProc.hpp"
 
 using std::shared_ptr;
 
+proc_t vfs_context_proc(vfs_context_t ctx)
```

Comment:
@pmj this was annoying but when I move this function to MockProc, the linker complained about a dual definition.  Do you see any obvious problems here?

Reply:
Hmm, I don't know what might have been the problem there. I just tried it and it worked straight away:

https://github.com/pmj/VFSForGit/commit/98d80aa41262d47efcb05ffa700f0552951c1675

Reply:
Thanks for trying that @pmj, yeah it seems totally fine now.  Perhaps I was working too late :)